webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -46,8 +45,8 @@ const Breadcrumb = ({
           <BreadcrumbItem
             key={index}
             isActive={active}
-            onClick={item.onClick}
-            to={item.url}
+            onClick={onClick}
+            to={url}
             className={classNames('breadcrumb-item', {
               active,
               'breadcrumb-item-with-icon': icon && active,
```

Comment:
Unrelated note, but I can't comment there. Why is there a regular space between an icon and the inner text? Shouldn't that be a non-breaking space so they stick together?

Reply:
It is valid comment, but unrelated to this PR, will merge without it.